index.js

```diff
@@ -88,6 +87,19 @@ function getData(swagger, path, operation, response, config, info) {
     data.pathParams = config.pathParams;
   }
 
+  // used for checking inputTesting table
+  var tempPath = (((swagger.basePath !== undefined) &&
+      (swagger.basePath !== '/'))
+      ? swagger.basePath : '') + path;
+
+  // get inputTesting from config if defined for this path:operation:response
+  if (config.inputTesting &&
```

Comment:
woudn't it make more sense to users to call the config entry something like 'request-data' instead of inputTesting?